projects/plugins/wpcomsh/wpcomsh.php

```diff
@@ -50,7 +50,6 @@
 /**
  * WP.com Widgets (in alphabetical order)
  */
-require_once __DIR__ . '/widgets/class-aboutme-widget.php';
```

Comment:
This widget contains the following warning, I think it's safe to remove!
<img width="662" alt="Screenshot 2025-04-10 at 17 33 07" src="https://github.com/user-attachments/assets/c7f7110d-5a6a-42ba-9d05-9a609c3d0d10" />